src/main/java/no/unit/nva/cristin/projects/UriUtils.java

```diff
@@ -31,6 +32,7 @@ public static String queryParameters(Map<String, String> queryParameters) {
 
     private static String formatQueryParameters(Map<String, String> queryParams) {
         return queryParams.entrySet().stream()
+            .sorted(Entry.comparingByKey())
```

Comment:
Since we're sorting, is it better to change the argument of the method to a SortedMap and pass a TreeMap that orders the keys by natural key order?